apps/judicial-system/web/src/routes/Shared/Requests/ActiveRequests.tsx

```diff
@@ -165,12 +165,23 @@ const ActiveRequests: React.FC<Props> = (props) => {
               user?.role && onRowClick(c.id)
             }}
           >
-            <td className={styles.td}>
-              <Text as="span">{c.policeCaseNumber || '-'}</Text>
-            </td>
+            {c.courtCaseNumber ? (
```

Comment:
Both the if and the else use 

`<td className={styles.td}>`

so I think it would be better to place the if/else statement inside the <td> tag.